src/main/resources/static/notebook_login.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script src="https://apis.google.com/js/platform.js?onload=renderButton" async defer></script>
+
+        <meta name="google-signin-client_id" content="$(defaultClientId)">
```

Comment:
We tested this and it got substituted as:
```
<meta name="google-signin-client_id" content=""806222273987-2ntvt4hnfsikqmhhc18l64vheh4cj34q.apps.googleusercontent.com"">
```
And gapi threw an error because it interpreted the client id as an empty string: "". Need to investigate why this field is coming through with extra quotes.

Reply:
It's because `template` explicitly quotes the replaced string. Will change this file to not include quotes.